implementations/micrometer-registry-dynatrace/src/main/java/io/micrometer/dynatrace/v1/DynatraceExporterV1.java

```diff
@@ -279,4 +282,22 @@ private Meter.Id idWithSuffix(Meter.Id id, String suffix) {
         return id.withName(id.getName() + "." + suffix);
     }
 
+    /**
+     * Redacts the API token from a thrown exception before printing the exception
+     * message.
+     * @param t the original {@link Throwable}
+     * @return the original {@link Throwable} if it does not contain the API token or a
+     * new {@link Throwable} with the message with the token redacted and same stack trace
+     * as the original Throwable.
+     */
+    private Throwable redactToken(Throwable t) {
+        if (t.getMessage() != null && !t.getMessage().contains(config.apiToken())) {
+            return t;
+        }
+
+        Throwable throwable = new Throwable(t.getMessage().replace(config.apiToken(), "<redacted>"));
```

Comment:
This would not retain information about the type of exception that has occured, right? :thinking: 
If not, would it make sense to add this info somehow? :thinking: